autoscale/src/main/resources/logback.xml

```diff
@@ -7,7 +7,7 @@
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
             <layout class="com.sequenceiq.cloudbreak.logger.MaskingPatternLayout">
-                <pattern>%date{ISO8601} [%thread] %method:%line %-5level %logger{5} - [type:%X{resourceType:-springLog}] [crn:%X{resourceCrn:-}] [name:%X{resourceName:-}] [tenant:%X{tenant:-}] [userCrn:%X{userCrn:-}] [requestid:%X{requestId:-}]  [traceId:%X{traceId:-}] [spanId:%X{spanId:-}] %msg%n</pattern>
+                <pattern>%date{ISO8601} [%thread] %method:%line %-5level %logger{5} - [type:%X{resourceType:-springLog}] [crn:%X{resourceCrn:-}] [name:%X{resourceName:-}] [tenant:%X{tenant:-}] [userCrn:%X{userCrn:-}] [requestid:%X{requestId:-}]  [traceId:%X{traceId:-}] [spanId:%X{spanId:-}] ::: %msg%n</pattern>
```

Comment:
As I see this change has nothing to do with the main feature which is already a huge change. Please create a separate PR for it but I'm not sure it is even needed. We have a common logging format in Cloudbreak why does it need to be different?

Reply:
Will roll this back.

Reply:
Fixed in the next push. Resolving conversation.